package/tests/test_models/test_graph/test_graph_nodes.py

```diff
@@ -426,10 +426,30 @@ def test_plotly_json_dataset_node_metadata(self, patched_json_load):
         patched_json_load.return_value = mock_plot_data
         plotly_json_dataset_node = MagicMock()
         plotly_json_dataset_node.is_plot_node.return_value = True
-        plotly_json_dataset_node.is_metric_node.return_value = False
+        plotly_json_dataset_node.is_image_node.return_value = False
+        plotly_json_dataset_node.is_tracking_node.return_value = False
         plotly_node_metadata = DataNodeMetadata(data_node=plotly_json_dataset_node)
         assert plotly_node_metadata.plot == mock_plot_data
 
+    @patch("base64.b64encode")
+    def test_image_data_node_metadata(self, patched_base64):
```

Comment:
Great job writing a test for this function, which is definitely not easy to do! But even better would be if it's possible to patch `img_file.read`'s return value instead? Maybe there's even some real (very short) image string we could put in here to make it a bit more a real test?

Reply:
Surely. I've added a very small base64 PNG to the test, taken from [here](https://stackoverflow.com/a/36610159).